pkg/bootstrap/node.go

```diff
@@ -628,3 +629,41 @@ func ensureIptables() {
 		}
 	}
 }
+
+func isSELinuxEnforcing() bool {
+	var cmdGetPath string
+	var err error
+
+	if cmdGetPath, err = exec.LookPath("getenforce"); err != nil {
+		// fall back to an ordinary abspath
+		cmdGetPath = "/usr/sbin/getenforce"
+	}
+
+	argsGet := []string{
+		cmdGetPath,
+	}
+
+	cmdGet := exec.Cmd{
+		Path:   cmdGetPath,
+		Args:   argsGet,
+		Env:    os.Environ(),
+		Stderr: os.Stderr,
+	}
+
+	// As getenforce always returns non-error, we should ignore the error.
+	// Instead, parse the output string directly to determine the current
+	// SELinux mode.
+	outstr, _ := cmdGet.Output()
+	sestatus := strings.TrimSpace(string(outstr))
+	if sestatus == "Enforcing" {
+		return true
+	}
+
+	return false
+}
+
+func ensureSelinux() {
+	if isSELinuxEnforcing() {
+		log.Fatalln("ERROR: SELinux enforcing mode is enabled. exit.")
```

Comment:
Should we not guide the use as to what to do? For example, "SELinux enforcing mode is enabled. you will need to disable it with `sudo setenforce 0` for kube-spawn to work properly."

In general, let's always guide the user if we can.